raj_ag.json

```diff
@@ -0,0 +1,10 @@
+{
+    "title" : "Doraemon",
+    "image" : "https://i.pinimg.com/236x/cf/a4/75/cfa475ac29e6a259efa430dc0a81225a.jpg",
+    "description" :" A cat-like robot of the 22nd century, He loves Japanese traditional sweet Dorayaki and suffers from musophobia",

```

Comment:
Added a JSON file with details for new superhero- Doraemon. Code structure and indent is at par. Pull request spelling is wrong.